test-suite/squidconf/bad-acl-dstdomain-dupe.conf

```diff
@@ -6,7 +6,7 @@
 ##
 
 acl test11 dstdomain .example.com example.com
-acl test12 dstdomain example.com .example.com
+acl test12 dstdomain example.org .example.org
```

Comment:
We slightly changed these and similar test cases to make their WARNING lines unique (after those lines lost the ACL name that was moved into another/dedicated `acl name: ...` line).